recipes-support/libunwind/libunwind_1.1.bbappend

```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# Add necessary patches according to:
+# https://github.com/dotnet/coreclr/blob/master/Documentation/building/linux-instructions.md#build-for-armlinux
+
+SRC_URI_append += "\
+           file://libunwind.git-770152268807e460184b4152e23aba9c86601090.patch \
+           file://expand-arm-Validate-memory-before-access.patch \
+"
```

Comment:
Ah, I just saw you are actually running on krogoth...

Is there a way to make this optional? When I use this as `libunwind_%.bbappend`, my libunwind build on morty breaks with `../../git/src/arm/Ginit.c:134:1: error: redefinition of 'validate_mem'`